src/XrdHttp/CMakeLists.txt

```diff
@@ -20,6 +20,11 @@ add_library(XrdHttpUtils SHARED
   XrdHttpHeaderUtils.cc      XrdHttpHeaderUtils.hh
 )
 
+target_sources(XrdHttpUtils PRIVATE
+  XrdHttpMon.hh
+  XrdHttpMon.cc
+)
```

Comment:
You can simply list the two extra source files as part of the `add_library(XrdHttpUtils SHARED ...)` command above.